internal/kafka/internal/workers/cluster_mgrs/deprovisioning_cluster_mgr.go

```diff
@@ -0,0 +1,164 @@
+package cluster_mgrs
+
+import (
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/kafka/internal/config"
+
+	"sync"
+
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/kafka/internal/services"
+	fleeterrors "github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/errors"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/workers"
+	"github.com/goava/di"
+	"github.com/google/uuid"
+
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/api"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/metrics"
+	"github.com/golang/glog"
+
+	"github.com/pkg/errors"
+)
+
+const (
+	deprovisioningWorkerType = "cluster_deprovisioning"
+)
+
+// DeprovisioningClusterManager represents a cluster manager that periodically reconciles data plane clusters in deprovisioning state.
+type DeprovisioningClusterManager struct {
+	id           string
```

Comment:
Same comment as https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1266/files#r957100730

Reply:
adjusted in the latest commit